### DISCUSSION TITLE
Error on importing torch in local system (ImportError: cannot import name 'classproperty' from 'torch._utils' (C:\Users\Administrator\anaconda3\envs\envpytorch2\lib\site-packages\torch\_utils.py))

### DISCUSSION THREAD

**Discussion by deepraj-7** [mrdbourke/pytorch-deep-learning#312] [Q&A]
I have no gpu and I have installed pytorch for cpu and after installing, I get the following error while importing:

`ImportError: cannot import name 'classproperty' from 'torch._utils' (C:\Users\Administrator\anaconda3\envs\envpytorch2\lib\site-packages\torch\_utils.py)`

What's the solution to this?
Thanks in advance.

**Comment by mrdbourke** [comment]
Hmm, this may be a question for the PyTorch forums, see here: https://discuss.pytorch.org/

Have you tried uninstalling PyTorch and reinstalling it using the page here: https://pytorch.org/get-started/locally/

Steps to uninstall:

```
pip3 uninstall torch torchvision torchaudio
```

Steps to install:

```
pip3 install torch torchvision torchaudio
```
